### PR TITLE
[batch] add estimated current SQL as documentation

### PR DESCRIPTION
**Pull request by cseed**
I liked your diff idea, so I added a new file: batch/sql/estimated-current.txt.  This is meant to be the SQL we'd use for initial.sql if we recreated the batch database.  It should have collective migrations applied to it.  So when we add a new migration, we should update the estimated current which will give informative documentation for the current change.  I use "estimated" and "txt" because it isn't tested or validated in any way.

### CODE REVIEW

File: batch/sql/estimated-current.txt [hunk 1]
```diff
@@ -0,0 +1,678 @@
...
+  IN in_instance_name VARCHAR(100),
+  IN in_reason VARCHAR(40),
+  IN in_timestamp BIGINT
+)
+BEGIN
+  DECLARE cur_state VARCHAR(40);
+
+  START TRANSACTION;
+
+  SELECT state INTO cur_state FROM instances WHERE name = in_instance_name FOR UPDATE ;
```
Comment (jigold) [inline batch/sql/estimated-current.txt]: Space at the end of this line.

**Comment by jigold** [comment]
You have a file naming error in build.yaml.

**Comment by cseed** [comment]
So it turns out jobs.cancelled is used in a non-trivial way.  So I backed out the change to drop the cancelled column, and just left in the estimated-current.txt.

**Comment by cseed** [comment]
bump.  This is just documentation now, and if I rebase the cancel PR on it, may make a little easier to parse.

**Comment by jigold** [comment]
Can you change the title first?

**Comment by cseed** [comment]
Done

**Pull request merged**